.github/actions/setup-node-ci/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Setup Node for REST API CI
+description: Uses specified Node version and runs npm commands to set up the environment for REST API CI
+
+inputs:
+  node-version:
+    description: 'Node version to use'
```

Comment:
```suggestion
```

Can we remove the description?

Reply:
Schema requires a description. We should keep those to avoid schema validation errors in editors at the least. I didn't directly test whether workflows will run without it but if it is allowed, schema validation could change in the future and it may become required. 

![image](https://github.com/user-attachments/assets/a3cf1c35-53e4-4f0d-88af-a7ede1d8906a)